mbf_abstract_nav/src/abstract_planner_execution.cpp

```diff
@@ -312,43 +308,33 @@ void AbstractPlannerExecution::run()
           // old nav_core-based planners do not support canceling), and we add here the fact to the log for info
           ROS_INFO_STREAM("Planning patience (" << patience_.toSec() << "s) has been exceeded"
                                                 << (cancel_ ? "; planner canceled!" : ""));
-          exceeded = true;
-          planning_ = false;
           setState(PAT_EXCEEDED, true);
         }
         else if (max_retries_ == 0 && patience_.isZero())
         {
           ROS_INFO_STREAM("Planning could not find a plan!");
-          exceeded = true;
-          planning_ = false;
           setState(NO_PLAN_FOUND, true);
         }
         else
         {
-          exceeded = false;
+          planning_ = true;
```

Comment:
we used to set planning_ = false in all cases but this. Now I set it once to false and change it here to true to reduce verbosity